kestest/gateway_test.go

```diff
@@ -633,3 +626,28 @@ func mustDecodeB64(s string) []byte {
 	}
 	return b
 }
+
+func clean(ctx context.Context, client *kes.Client, pattern string, t *testing.T) {
```

Comment:
You can omit the pattern here and use the match all pattern `*` in `ListKeys`.

Reply:
> You can omit the pattern here and use the match all pattern `*` in `ListKeys`.

Ok, makes sense. Will do the changes.